java/integTest/src/test/java/glide/cluster/ClusterClientTests.java

```diff
@@ -111,7 +111,7 @@ public void send_and_receive_non_ascii_unicode() {
     @ParameterizedTest
     @ValueSource(ints = {100, 2 ^ 16})
     public void client_can_handle_concurrent_workload(int valueSize) {
-        ExecutorService executorService = Executors.newFixedThreadPool(8);
```

Comment:
Should be run test with different thread pool types?
You can parametrize the tests by
* thread pool type
* whether it is shared with client or not

Reply:
Changed this to cached thread pool which actually is the better test here. (Also replicates benchmarks)

Reply:
A user may want to use any type of the pool

Reply:
We can't support every pool for example our benchmark, it would fail at ForkJoinCommonPool when too many threads were created. This test is just to test the concurrency.